#Configuation for robot odometry EKF
#
jack_hybobs_subIMU: "/imu/data"
jack_hybobs_subOdom: "/DT0/gradientOdom"
jack_hybobs_pubOdom: "/HYB0/Odom"
jack_hybobs_pubIMU: "/HYB0/IMU"
jack_hybobs_pubBias: "/HYB0/Bias"
DistjackAPIName: "HYB0API"
NanchorMesh: 4
HYBrate: 50